solidity/package.json

```diff
@@ -1,15 +1,19 @@
 {
 	"version": "1.0.0",
+	"type": "module",
 	"description": "Augur-Constant-Product-Market-Contracts",
 	"scripts": {
-		"compile": "tsc && node ./compile.js && node --input-type=module -e \"import { promises as fs } from 'fs'; await fs.rm('./compile.js')\""
+		"compile": "tsc && node ./js/compile.js",
```

Comment:
We should switch to `node test`.  This will give us a proper test runner, with aggregate test output, parallelization (if we want), ability to run individual tests from command line, etc.  https://nodejs.org/docs/latest-v22.x/api/test.html